Namco/pacman.py

```diff
@@ -91,30 +90,30 @@ def create_observation(self):
     def move_pacman(self, action):
         # Moving up
         if action == 'w':
-            self.x = max(self.x - 1, 0)
+            self.pacman[0] = max(self.pacman[0] - 1, 0)
         # Moving left
         elif action == 'a':
-            self.y = max(self.y - 1, 0)
+            self.pacman[1] = max(self.pacman[1] - 1, 0)
         # Moving down
         elif action == 's':
-            self.x = min(self.x + 1, self.map_size - 1)
+            self.pacman[0] = min(self.pacman[0] + 1, self.map_size - 1)
         # Moving right
         elif action == 'd':
-            self.y = min(self.y + 1, self.map_size - 1)
+            self.pacman[1] = min(self.pacman[1] + 1, self.map_size - 1)
 
     def generate_objects(self):
         for _ in range(self.num_of_objects):
             obj_coords = tuple(np.random.randint(0, self.map_size, (2,)))
 
             # Making sure that generated objects do not have the same position with each other or with Pacman
-            while (obj_coords in self.objects) or (obj_coords in tuple([self.x, self.y])):
+            while (obj_coords in self.objects) or (obj_coords in tuple(self.pacman)):
```

Comment:
(obj_coords in tuple(self.pacman):
Pacman egy tuple, nem egy lista, így hibásan fog működni.

Reply:
Igen, köszi szépen! Javítottam ==-re